src/components/NFTEditionSelectItem.vue

```diff
@@ -52,7 +81,26 @@
         cellClasses,
       ]"
     >
-      {{ priceLabel }}
+      <div class="flex items-center justify-end w-full gap-[12px]">
+        <div
+          v-if="discountInfo"
+          class="flex justify-end items-center gap-[4px]"
+        >
+          <span
+            class="text-shade-gray line-through font-400 text-[12px] text-right"
+          >
+            {{ discountInfo.originalPriceLabel }}
+          </span>
+          <span class="text-danger font-400 text-[12px] text-right">
```

Comment:
Same as above

Reply:
[💄 Improve clarity of discount text](https://github.com/likecoin/liker-land/pull/1544/commits/f264b16fb40e8fa61c6a8551d6e29de32dd7e6ac)

Reply:
[💄 Improve clarity of discount text](https://github.com/likecoin/liker-land/pull/1544/commits/f264b16fb40e8fa61c6a8551d6e29de32dd7e6ac)